host/bcm283x/dma.go

```diff
@@ -823,6 +1113,16 @@ func (d *driverDMA) Close() error {
 	return nil
 }
 
+func debugDMA() {
+	for i, ch := range dmaMemory.channels {
+		fmt.Println(i, ch.cs.String())
+		if ch.cs&dmaActive != 0 {
+			fmt.Printf("%x: %s", ch.cbAddr, ch.GoString())
```

Comment:
I'd prefer log instead of fmt

Reply:
Fixed in https://github.com/google/periph/pull/209.